app/global-error.tsx

```
@@ -0,0 +1,64 @@
+  reset,
+}: {
+  error: Error & { digest?: string };
+  reset: () => void;
+}) {
+  useEffect(() => {
+    // Log the error to an error reporting service
+    console.error('Global error:', error);
+  }, [error]);
+
+  return (
+    <html>
```

Biome (1.9.4)

[error] Provide a lang attribute when using the html element.

Setting a lang attribute on HTML document elements configures the languageused by screen readers when no user default is specified.

(lint/a11y/useHtmlLang)